tests/test.py

```diff
@@ -128,6 +128,20 @@ def fetch_json(self, path, params={}, **kwargs):
 # TODO: split the handler tests into another file
 
 
+class TestRootHandler(BaseTestCase):
+    def test_home_page(self):
+        response = self.fetch('/')
+        File = open("index.html", "r")
+        if File.mode == 'r':
+            contents =File.read()
+        documentation = b"http://wiki.apertium.org/wiki/Apertium-apy"
+        source = b"https://github.com/apertium/apertium-apy"
+        if documentation in response.body and source in response.body:
```

Comment:
Please decode the response body and then save it to a variable. Instead of `assertTrue(True)` etc, just put the expression you're interested in testing inside the assert (see examples below).